Community/committer-list.md

```diff
@@ -7,3 +7,12 @@
 |<a href="https://github.com/xuguruogu" target="_blank"><img src="https://avatars.githubusercontent.com/u/6092236" width="50" height="50"></a>| [kent](https://github.com/xuguruogu)   |  BENLI LI   | Tencent      |
 | <a href="https://github.com/datian9966" target="_blank"><img src="https://avatars.githubusercontent.com/u/47170471" width="50" height="50"></a> |[chenxu14](https://github.com/chenxu14) | Chen Xu | Meituan |
 | <a href="https://github.com/guojun85" target="_blank"><img src="https://avatars.githubusercontent.com/u/5196917" width="50" height="50"></a> |[Jun Guo](https://github.com/guojun85) |  | Bigo |
+
+
+### vesoft-inc/nebula-go Committers
```

Comment:
we cannot miss https://github.com/haoxins and/or https://github.com/peczenyj ?

Reply:
Of course, I will formally initiate the voting process for them to become maintainers and committers of the project.

Reply:
If you join peczenyj and haoxins after voting, please mark the conversation as resolved